.github/workflows/ci.yml

```diff
@@ -1,10 +1,10 @@
 name: CI
 
 on:
-  push:
-    branches: [ main ]
+  push: 
+    branches: [ master ]
```

Comment:
We already changed to main, I think it's no longer necessary.

cc @ajsantander @mjlescano